sylvia-derive/src/message.rs

```diff
@@ -247,7 +249,7 @@ impl<'a> EnumMessage<'a> {
                     };
 
                     if attr == ty {
-                        Some(MsgVariant::new(&method.sig, &mut generics_checker))
+                        Some(MsgVariant::new(&method.sig, &mut generics_checker, name))
```

Comment:
Naaaaaaah, you don't pass the name - the message still should be named as it was. Only an alias for gluing.